### ISSUE TITLE
OpenBAS agent failed to install on windows machine

### ISSUE THREAD

**Issue by marckto-filigran** [OpenBAS-Platform/openbas#1151]
## Description

There is an error with the installation script.

## Environment

OS: win 10 x86_64
OpenBAS version: 1.1.0
https://demo.obas.filigran.io/

## Reproducible Steps

Steps to create the smallest reproducible scenario:
1. Copy the installation command to windows cmd (admin mode)

## Expected Output

agent is installed

## Actual Output
 
Error
![image](https://github.com/OpenBAS-Platform/openbas/assets/166409476/0a1a49ac-c513-4447-be02-c86df964fc1a)

## Additional information
I managed to install the agent by downloading it manually then executing the powershell commands separately

**Comment by savacano28** [comment]
Tested locally from demo, and it works well. Waiting for Mark's point to proceed with testing.

**Comment by savacano28** [comment]
I discussed with Mark, and the host system has outdated settings and the architecture is 32-bit, which is not supported by the agent.

**Comment by savacano28** [comment]
Actions:
1. Update powershell: OK
2. Create folder for agent : OK
3. Create service : OK

Next time
5. Execute command iwr from demo

**Comment by savacano28** [comment]
Actions (07-22-24 with Mark):
Guide: https://www.notion.so/filigran/OpenBAS-Community-a276855e122b480cadacbb47133a508a
We're still encountering errors when we try to install the agent on the Windows VM, possibly due to conflicts with services that are either not stopped or do not exist.

**Comment by savacano28** [comment]
Actions (19-08-24):  
I got access to a copy of Mark's VM and was able to install the OpenBas agent. The procedure was as follows: I created a folder at C:\Program Files (x86)\Filigran and used this path to create a service named 'OpenBas Agent Service'. Then, I used the following command to retrieve all the files for the OpenBas agent: iex (iwr "https://testing.obas.staging.filigran.io/api/agent/installer/openbas/windows/8aa00a10-9d01-4ca8-aa28-e225923f65f2").Content. After that, I was able to locate the machine on openbas-staging/endpoints/ and execute a custom payload. 

![image](https://github.com/user-attachments/assets/39575e77-8cce-406f-ab53-64e4cf041295)

![image](https://github.com/user-attachments/assets/40838538-357c-4eaf-af7b-7961a2ddee1c)

![image](https://github.com/user-attachments/assets/4c2ee4ef-ac78-4dd8-947a-496884deee15)
